### DISCUSSION TITLE
Add sponsor button

### DISCUSSION THREAD

**Discussion by blanklob** [EvgeniyMukhamedjanov/liquid-ajax-cart#83] [General]
# Add a sponsor button so we can sponsor man!

**Comment by EvgeniyMukhamedjanov** [comment]
Hahaha :-) 
Thank you for this, so heartwarming.
Right now not many people know about Liquid Ajax Cart, so I would prefer sharing in social media or GitHub stars as a contribution rather than donates :-) 

Btw if you make a website using the library, I can add it to the liquid-ajax-cart.js.org home page into the Showcases section. Let me know :-)

**Reply by blanklob** [reply]
Just do it through github man, I will sponsor.

**Reply by EvgeniyMukhamedjanov** [reply]
Waiting for GitHub approval :-)
